docs/sources/configure-server/about-server-api.md

```diff
@@ -0,0 +1,196 @@
+---
+description: Learn about the Pyrocope server API
+menuTitle: Server API Overview
+title: Pyroscope Server HTTP API Reference
+weight: 20
+---
+
+# Pyroscope Server HTTP API Reference
+
+Pyroscope server exposes an HTTP API that can be used to query profiling data and to ingest profiling data from other sources.
+
+## Ingestion
+
+Currently there's just one endpoint: `POST /ingest`. It takes profile data in request body and metadata as query params.
+
+It takes multiple query parameters:
+
+| Name               | Description                             | Notes                           |
+|:-------------------|:----------------------------------------|:--------------------------------|
+| `name`             | application name                        | required |
+| `from`             | UNIX time of when the profiling started | required                        |
+| `until`            | UNIX time of when the profiling stopped | required                        |
+| `format`           | format of the profiling data            | optional (default is `folded`)  |
+| `sampleRate`       | sample rate used in Hz                  | optional (default is 100 Hz)    |
+| `spyName`          | name of the spy used                    | optional                        |
+| `units`            | name of the profiling data unit         | optional (default is `samples`  |
+| `aggregrationType` | type of aggregration to merge profiles  | optional (default is `sum`)     |
+
+* `name` specifies application name. It may be specified in the fully qualified application name notation as described in [the specification](/docs/flameql#notation). For example:
+```
+my.awesome.app.cpu{env=staging,region=us-west-1}
```

Comment:
how does this work now with `{service_name="xyz"}`

Reply:
Our `/ingestion` API won't be able to handle labels in the new format.

I think it's better to add a new parameter (to avoid clashes with the old logic), like `labels` and not to mention `name` and flameql here as to not confuse users. We will handle both params it in the backend, but only `labels` should be part of the public API